docs/release-process/instructions.md

```diff
@@ -0,0 +1,85 @@
+# How to use the release tooling
+
+## Creating the tracking issue
+
+First, create an issue on the microsoft/go repository for release progress updates. When you pass the issue number to the release automation pipeline, that makes the pipeline post a comment on that issue to notify you when a step completes or fails.
+
+If dealing with multiple releases at the same time (for example, 1.17 and 1.18 security patches released on the same day), create one issue per version number. This makes it easier to keep track of each release's progress as an outside observer.
+
+> The "release-start" pipeline is planned to create these issues automatically for common servicing events. (See [README.md](README.md).)
+
+## Upstream release
+
+After an upstream release, or in anticipation of an upstream release, go to the [`microsoft-go-infra-release-build`](https://dev.azure.com/dnceng/internal/_build/results?buildId=1780270&view=results) pipeline. Press "Run new" and fill in the blanks. For example, a 1.42.3-1 patch release would look like this:
+
+> ![](images/run-upstream-release.png)
```

Comment:
Where do you end up explaining the purpose of the other parameters here?

Reply:
Hmm, I was thinking the field titles would explain it fine, but the numbered ones are definitely not clear at this point in the doc. I'll add a description.

Do you think the version number, issue number, and variable group ones are clear enough from the screenshot?

Reply:
Added a list below the screenshot explaining a little more.